.gitignore

```diff
@@ -0,0 +1,2 @@
+result
+result-*
```

Comment:
Early merging this file in https://github.com/danth/stylix/pull/291 benefits the project right away. The merge commit would remove this file:

```suggestion
```

Reply:
#291 is now merged